src/unitxt/processors.py

```diff
@@ -175,6 +175,7 @@ def process_value(self, text: Any) -> Any:
 
 
 class MatchClosestOption(InstanceFieldOperator):
+    __description__ :  str = "Note: this post processor only works with MultipleChoiceTemplate"
```

Comment:
I think this can be removed.  This should work with any task with options in it's task data.